tests/subsys/portability/cmsis_rtos_v2/src/event_flags.c

```diff
@@ -217,3 +217,10 @@ void test_event_flags_isr(void)
 	zassert_true(osEventFlagsDelete(evt_id) == osOK,
 		     "EventFlagsDelete failed");
 }
+ZTEST(cmsis_event_flags, test_event_flags)
+{
+	test_event_flags_no_wait_timeout();
```

Comment:
Are these here in one ZTEST because they're order-dependent? If so, we should comment as such.

Reply:
Yes, they are order-dependent. And I will comment it in code. After that, please have a look again. Thanks.

Reply:
@aaronemassey Hi, please have a look again. Thanks.

Reply:
Hi @shuai1x , I wonder this three tests could be independent. Could you please try this: add "osEventFlagsNew()" for another two of them, then do "osEventFlagsClear()" at last line for all them. (If it works, I think it also can be put in setup/teardown function.) To see if this can be run independently. Thanks.